src/logging/monitor.ts

```diff
@@ -144,6 +153,50 @@ class LogMonitor {
 		void this.start();
 	}
 
+	private async getJournalctlOptions(): Promise<SpawnJournalctlOpts> {
+		try {
+			// On SV start, journalctl is spawned with a timestamp to only
+			// get logs since the last Supervisor State.FinishedAt. This will catch any
+			// host and container logs generated while the Supervisor was not running.
+			const supervisorContainer = await getSupervisorContainer();
+			if (supervisorContainer !== null) {
+				const finishedAt = supervisorContainer.State.FinishedAt;
+				const finishedAtDate = new Date(finishedAt).getTime();
+				// When a container has never exited with any exit code,
+				// the FinishedAt timestamp is "0001-01-01T00:00:00Z". Any
+				// timestamp below 0 in ms value is invalid.
+				// Only set the lastSentTimestamp to the last Supervisor State.FinishedAt if:
+				// - finishedAtDate is greater than 0 (i.e. the supervisor container has exited at least once)
+				// - lastSentTimestamp is null (i.e. this is the first time we've started the monitor)
+				//   - This prevents the case of the logs getting streamed from State.FinishedAt for
+				//     for subsequent monitor.start() calls due to the underlying journalctl process dying.
+				if (finishedAtDate > 0 && this.lastSentTimestamp == null) {
+					this.lastSentTimestamp = finishedAtDate;
+				}
+			}
+		} catch {
+			// Ignore errors getting the supervisor container
```

Comment:
Is this catch necessary since you are catching errors in the `getSupervisorContainer` function?

Reply:
True, removed this try/catch